lib/teleterm/clusters/cluster_gateways.go

```diff
@@ -71,7 +71,10 @@ func (c *Cluster) createDBGateway(ctx context.Context, params CreateGatewayParam
 		Username:    params.TargetUser,
 	}
 
-	if err := c.reissueDBCerts(ctx, routeToDatabase); err != nil {
+	err = AddMetadataToRetryableError(ctx, func() error {
+		return trace.Wrap(c.reissueDBCerts(ctx, routeToDatabase))
+	})
```

Comment:
This is the same thing that I've done in #35355, but this time for db gateways. When working on #35355, I didn't realize that db gateways were using `AddMetadataToRetryableError` this way too.